rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -174,6 +174,68 @@ TEST_F(RosBag2PlayTestFixture, recorded_messages_are_played_for_all_topics)
           ElementsAre(40.0f, 2.0f, 0.0f)))));
 }
 
+TEST_F(RosBag2PlayTestFixture, recorded_messages_are_played_for_all_topics_multi)
```

Comment:
```suggestion
TEST_F(RosBag2PlayTestFixture, recorded_messages_are_played_for_all_topics_from_three_bags)
```

Reply:
changed in fd349cc793eab28feb6ef089c20ad1a6329a4001